### ISSUE TITLE
Various payment list is white page on v18.0.1

### ISSUE THREAD

**Issue by citananda** [Dolibarr/dolibarr#25869]
### Bug

the page for various payments appears empty 

### Environment Version

18.0.1

### Environment OS

Debian 10.13

### Environment Web server

Apache/2.4.38

### Environment PHP

PHP 8.1.20

### Environment Database

mariadb from 11.1.2-MariaDB, client 15.2 for debian-linux-gnu (x86_64)

### Environment URL(s)

/compta/bank/various_payment/list.php

### Expected and actual behavior

We are using the Docker version tuxgasy/dolibarr:latest with mariadb:latest.
After updating to Dolibarr version 18.0.1, the page for various payments appears empty (/compta/bank/various_payment/list.php).
Upon inspecting the logs, the display of this page generates the following logs:

```
PHP Warning:  Undefined variable $toselect in /var/www/html/compta/bank/various_payment/list.php on line 370
PHP Warning:  Undefined variable $arrayofmassactions in /var/www/html/compta/bank/various_payment/list.php on line 427
PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /var/www/html/core/class/html.form.class.php:830\nStack trace:\n#0 /var/www/html/compta/bank/various_payment/list.php(427): Form->selectMassAction()\n#1 {main}\n  thrown in /var/www/html/core/class/html.form.class.php on line 830
GET /compta/bank/various_payment/list.php?leftmenu=tax_various&mainmenu=billing HTTP/1.1" 200
```

### Steps to reproduce the behavior

Update from 17.0.2 to 18.0.1

**Comment by ksar-ksar** [comment]
FIX : #25861

**Comment by citananda** [comment]
Perfect, thanks
